textile/src/main/java/io/textile/textile/Account.java

```diff
@@ -30,7 +30,7 @@ public String seed() {
 
     public Contact contact() throws Exception {
         byte[] bytes = node.accountContact();
-        return Contact.parseFrom(bytes);
+        return bytes != null ? Contact.parseFrom(bytes) : null;
```

Comment:
General thinking here is, even when a value is always expected to return data from go-textile, just make sure we have bytes to parse and return null otherwise. Just to be safe. If there is some underlying problem, go-textile will surface an Exception anyway.